### ISSUE TITLE
import bot

### ISSUE THREAD

**Issue by Olegt0rr** [mail-ru-im/bot-python#11]
Naming your library as a `bot` is a bad practice.
Not clear to users, not clear to dependencies.
Bot development is not just about you :)
A more suitable and clear name option is `icq`

**Comment by nightadmin** [comment]
You are right that "bot" is not the best solution, but the developers can name this library as "icq-bot", it would be easier to understand what the library is for than "icq" or "bot".
